### ISSUE TITLE
Bug report: connection refused error while encryption is turned off (HTTP mode)

### ISSUE THREAD

**Issue by notjedi** [localsend/localsend#218]
### Describe the bug

all the below observations were made with the `Encryption` setting turned off. the app versions are also the same b/w devices.

error message when trying to share files b/w an android phone and a linux pc:
```java
DioError [DioErrorType.other]: HandshakeException: Connection terminated during handshake
Source stack:
#0      DioMixin.fetch (package:dio/src/dio_mixin.dart:488)
#1      DioMixin.request (package:dio/src/dio_mixin.dart:483)
#2      DioMixin.post (package:dio/src/dio_mixin.dart:97)
#3      SendNotifier.startSession (package:localsend_app/provider/network/send_provider.dart:97)
<asynchronous suspension>
```

getting the same error when trying to share files b/w android devices.

### To reproduce

1. Go to settings and turn off Encryption.
2. Restart the server or restart the app.
3. do the same on the other device.
4. try to send some file over to the other device

### Expected behavior

should prompt the recipient about new incoming files.

### Desktop (please complete the following information)

- OS: Arch Linux
- Version: `1.7.0`

### Smartphone (please complete the following information)

- Device: tested on multiple android devices, namely: Poco M2 Pro and Moto G52.
- OS: Android
- Version: `1.7.0`

### Additional context

here are my findings while debugging this issue: 

even with Encryption turned off, the sender is sending `TLSv1` `Client Hello` packets to the recipient, to which the recipient "panics". `TLS` packets should not be sent in HTTP connections as far as i know, but i may be wrong here.

below is a screenshot of the wireshark dump(take a look at number 531 through 538):

![localsend](https://user-images.githubusercontent.com/30691152/221935551-f52c6582-889b-4af6-a147-cb96c0073a10.png)

**Comment by Tienisto** [comment]
Does the error still exist after you restarted the app on both devices? And encryption is turned off on both I guess?

**Comment by notjedi** [comment]
yes, i did restart the app on both devices

**Comment by Tienisto** [comment]
Fixed in `v1.8.0`
